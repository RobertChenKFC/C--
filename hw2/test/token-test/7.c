../assignment-test/7.c