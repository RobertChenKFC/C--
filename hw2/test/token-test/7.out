COMMENT
/***/
IDENTIFIER
IDENTIFIER
IDENTIFIER
IDENTIFIER
OP_MUL
OP_DIV

Frequency of identifiers:
be                               1
identifiers                      1
should                           1
these                            1
